experiment/build/docker_images.py

```diff
@@ -40,7 +44,29 @@ def _instantiate_image_obj(name_template, obj_template, fuzzer, benchmark):
 def _get_image_type_templates():
     """Loads the image types config that contains "templates" describing how to
     build them and their dependencies."""
-    return yaml_utils.read('docker/image_types.yaml')
+    all_templates = yaml_utils.read('docker/image_types.yaml')
```

Comment:
please use os.path.join instead of using '/'.
Also, let's use ROOT_DIR to find this file instead of assuming the working directory.